src/cli/cli_commissioner.cpp

```diff
@@ -334,6 +499,22 @@ template <> otError Commissioner::Process<Cmd("sessionid")>(Arg aArgs[])
     return OT_ERROR_NONE;
 }
 
+/**
+ * @cli commissioner id (get,set)
+ * @code
+ * commissioner id OpenThread commissioner
```

Comment:
OpenThread commissioner here

But the output below has Commissioner, they should match